### ISSUE TITLE
Circular Transistion Between TabBar Selections?

### ISSUE THREAD

**Issue by StormFactory** [Yalantis/FoldingTabBar.iOS#9]
In your sample video/gif, when you tapped on one of the tab bar items, it looked like there was a circular or radial transition from the bottom of the screen.  Is that part of this repository?

**Comment by TindleWei** [comment]
wanna know as well,  how to make that effect?

**Comment by maksTheAwesome** [comment]
The animated circular transition is not the part of the component but you are welcome to use the tutorial at the following link:

http://www.raywenderlich.com/86521/how-to-make-a-view-controller-transition-animation-like-in-the-ping-app

Please note that we are also open for pull request. Please feel free to share.

**Comment by TindleWei** [comment]
Awesome,  I also find this.
https://github.com/moqod/ios-material-design

**Comment by TindleWei** [comment]
Hi, Maks. I need help for making the circle transition.

I looked through
http://www.raywenderlich.com/86521/how-to-make-a-view-controller-transition-animation-like-in-the-ping-app
,

and try to change swift to object-c project,

When I use the Animation Controllers, problem occurred.

At using my MaterialCircleAnimator.m:

- (void)animateTransition:(id <UIViewControllerContextTransitioning
>)transitionContext

-(NSTimeInterval)transitionDuration:(id)transitionContext

I found my code can't change the animation duration time.

The circle animation would last 0.3 second no matter what NSTimeInterval I
set.

-----------------------------------------------------------------------------------------------------------------------

That's the refer project I download
http://cdn1.raywenderlich.com/wp-content/uploads/2014/12/CircleTransitionFinalUpdated.zip

And my project on Github:
https://github.com/TindleWei/MaterialCircleAnimation



2015-04-27 17:09 GMT+08:00 Maks <notifications@github.com>:
